test/setup.js

```diff
@@ -21,3 +15,13 @@ expect.extend({
     })
   }
 })
+
+if (!global.document) { // to make mocha's watch mode work
```

Comment:
before the 2nd run in watch mode the jsdom globals would be recreated and the fetch polyfill would get confused